ef-1: add endoint to return the timestamp

jungejason opened the pull request:
Summary:
AT

Test:
Access the service and it works:
```shell
╰─ http localhost:3000/timestamp
HTTP/1.1 200 OK
Connection: keep-alive
Content-Length: 13
Content-Type: text/html; charset=utf-8
Date: Wed, 09 Mar 2022 20:29:40 GMT
ETag: W/"d-ePhAByrLhGz5ArSY6UMvRcZ82Vc"
Keep-Alive: timeout=5
X-Powered-By: Express

1646857780950
```

jungejason committed:
ef-1: add endoint to return the timestamp

Summary:
AT

Test:
Access the service and it works:
```shell
╰─ http localhost:3000/timestamp
HTTP/1.1 200 OK
Connection: keep-alive
Content-Length: 13
Content-Type: text/html; charset=utf-8
Date: Wed, 09 Mar 2022 20:29:40 GMT
ETag: W/"d-ePhAByrLhGz5ArSY6UMvRcZ82Vc"
Keep-Alive: timeout=5
X-Powered-By: Express

1646857780950
```

jungejason2 reviewed (approved):
<img src="https://media2.giphy.com/media/Hw8vYF4DNRCKY/giphy.gif"/>

jungejason committed:
ef-3: get rid of hard-coded port number (#13)

* ef-3: get rid of hard-coded port number

Summary:
Use a variable instead of the hard-coded number.

Test:
1. `node index.js` to run the service
2. verified that the service is still working by

```shell
╰─ http localhost:3000
HTTP/1.1 200 OK
Connection: keep-alive
Content-Length: 11
Content-Type: text/html; charset=utf-8
Date: Wed, 09 Mar 2022 20:18:21 GMT
ETag: W/"b-Kq5sNclPz7QV2+lfQIuc6R7oRu0"
Keep-Alive: timeout=5
X-Powered-By: Express

hello world
```

* Add readme for doc

jungejason committed:
Merge branch 'master' into ef1-add-endpoint-for-ts
